### ISSUE TITLE
Error when adding multivectors

### ISSUE THREAD

**Issue by eschnett** [chakravala/Grassmann.jl#14]
I encounter this error:
```Julia
julia> basis"+++"
julia> (v1+v2) + (v1+v2)*(v1+v2)
ERROR: DimensionMismatch("No precise constructor for StaticArrays.MArray{Tuple{3},Int64,1,3} found. Length of input was 8.")
Stacktrace:
 [1] StaticArrays.MArray{Tuple{3},Int64,1,3}(::Tuple{Tuple{Tuple{NTuple{8,Int64}}}}) at /Users/eschnett/.julia/packages/StaticArrays/VyRz3/src/convert.jl:1
 [2] Type at /Users/eschnett/.julia/packages/StaticArrays/VyRz3/src/convert.jl:4 [inlined] (repeats 3 times)
 [3] convert at /Users/eschnett/.julia/packages/StaticArrays/VyRz3/src/convert.jl:10 [inlined]
 [4] value at /Users/eschnett/.julia/packages/Grassmann/LeGo1/src/multivectors.jl:327 [inlined]
 [5] +(::MBlade{Int64,⟨+++⟩,1,3}, ::MultiVector{Int64,⟨+++⟩,8}) at /Users/eschnett/.julia/packages/Grassmann/LeGo1/src/algebra.jl:1045
 [6] top-level scope at none:0
```

**Comment by chakravala** [comment]
Thanks for reporting the bug, it seems on line 1045 the `b` needs to be replaced by `a` and that fixes it.

**Comment by chakravala** [comment]
That bug should now be fixed on `master` branch, which will be v0.1.4 when released.
